docker-compose.override.private.yml

```
@@ -0,0 +1,66 @@
+  searxng:
+    ports:
+      - 127.0.0.1:8081:8080
+
+  ollama-cpu:
+    ports:
+      - 127.0.0.1:11434:11434
+
+  ollama-gpu:
+    ports:
+      - 127.0.0.1:11434:11434
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)